docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pandas_pq = BytesIO()
</pre><p>Don't declare variables until they are used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Dropped this.